modules/fedimint-mint-client/src/oob.rs

```diff
@@ -32,6 +32,7 @@ pub enum MintOOBStatesV1 {
 /// ```mermaid
 /// graph LR
 ///     Created -- User triggered refund --> RefundU["User Refund"]
+///     RefundU -- Refund failed --> RefundedMultiFallback["User Refund Fallback"]
```

Comment:
This isn't relevant anymore, right? Might be still worth mentioning the legacy and the new states.

Reply:
Leftover. Deleted.